qiita_db/metadata_template/util.py

```diff
@@ -262,32 +234,24 @@ def load_template_to_dataframe(fn, strip_whitespace=True, index='sample_name'):
     #   the values that should be considered "True" for boolean columns
     # false_values:
     #   the values that should be considered "False" for boolean columns
-    # converters:
-    #   ensure that sample names are not converted into any other types but
-    #   strings and remove any trailing spaces. Don't let pandas try to guess
-    #   the dtype of the other columns, force them to be a str.
     # comment:
     #   using the tab character as "comment" we remove rows that are
     #   constituted only by delimiters i. e. empty rows.
     try:
         template = pd.read_csv(
-            StringIO(''.join(holdfile)), sep='\t', encoding='utf-8',
-            infer_datetime_format=True, keep_default_na=False,
+            StringIO(''.join(holdfile)),
```

Comment:
Same comment as above - this is reading booleans, changing different NA values and this one is also parsing dates.